src/odemis/acq/fastem.py

```diff
@@ -837,26 +836,54 @@ def estimateTiledAcquisitionTime(stream, stage, area, dwell_time=None):
     """
     # get the resolution per tile used during overview imaging
     res = fastem_conf.SCANNER_CONFIG[fastem_conf.OVERVIEW_MODE]['resolution']
-    fov = fastem_conf.SCANNER_CONFIG[fastem_conf.OVERVIEW_MODE]['horizontalFoV']
+    fov_value = fastem_conf.SCANNER_CONFIG[fastem_conf.OVERVIEW_MODE]['horizontalFoV']
 
     # calculate area size
-    fov = (fov, fov * res[1] / res[0])
+    fov = (fov_value, fov_value * res[1] / res[0])
     acquisition_area = util.normalize_rect(area)  # make sure order is l, b, r, t
     area_size = (acquisition_area[2] - acquisition_area[0],
                  acquisition_area[3] - acquisition_area[1])
     # number of tiles
-    nx = math.ceil(abs(area_size[0] / fov[0]))
-    ny = math.ceil(abs(area_size[1] / fov[1]))
+    nx = math.ceil(abs(area_size[0] / fov[0]))  # Number of tiles horizontally
+    ny = math.ceil(abs(area_size[1] / fov[1]))  # Number of tiles vertically
 
-    # add some overhead per tile for e.g. stage movement
-    overhead = 1  # [s]
-    # time per tile
+    # Time for tile acquisition
     dwell_time_value = dwell_time if dwell_time is not None else stream.emitter.dwellTime.value
-    acq_time_tile = res[0] * res[1] * dwell_time_value + overhead  # [s]
-    # time for total overview image
-    acq_time_overview = nx * ny * acq_time_tile  # [s]
+    acq_time_tile = res[0] * res[1] * dwell_time_value
 
-    return acq_time_overview
+    # Total acquisition time for imaging (all tiles)
+    # add 2s to account for switching from one tile to next tile
+    # this time is added in TiledAcquisitionTask.estimateTime
+    acq_time = nx * ny * (acq_time_tile + 2)
```

Comment:
2s of additional overhead on top of the stage movement still sounds very high, did you also check on EA1 that this is a realistic value? If it is, do you understand where this is coming from? Is it for instance all in the overhead from transferring the images from the MPC to the Odemis PC?

Reply:
I checked on EA1 and this value is realistic. Honestly, I don't know where this overhead is coming from. There are a lot of futures and sub-futures in tiled acquisition. This 2s overhead apparently seems to work also for MIMAS and METEOR, it was added by @K4rishma in `TiledAcquisitionTask.estimateTime`

Reply:
Okay, let's keep it in mind them, maybe we want to invest this at some point